src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
+  font: var(--font-body-medium);
+  line-height: 1.3;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 15px;
+  max-width: 350px;
+  text-align: center;
```

Comment:
Since we generally want all text within this component to be center-aligned, you should just put this on the `.container` element and allow it to be inherited by child components.